go.mod

```diff
@@ -319,6 +320,7 @@ require (
 	github.com/xrash/smetrics v0.0.0-20201216005158-039620a65673 // indirect
 	github.com/zondax/hid v0.9.1 // indirect
 	github.com/zondax/ledger-go v0.12.1 // indirect
+	github.com/zyedidia/generic v1.2.1 // indirect
```

Comment:
This is a very small library with a bunch of very nice generic helpers. Wanted to use it since forever, finally got a quality excuse.